jetty9-base/pom.xml

```diff
@@ -112,7 +136,7 @@
               <goal>copy-dependencies</goal>
             </goals>
             <configuration>
-              <outputDirectory>${project.build.directory}/jetty-base/lib/gcp</outputDirectory>
+              <outputDirectory>${project.build.directory}/jetty-base/lib/gcloud</outputDirectory>
```

Comment:
I'm a bit confused. This is a change from current master branch. I just want to make sure it's intentional. Currently we copy all jars into  `/gcp`, but this will copy them to `/gcloud`.